ENCODE_is_complete.py

```diff
@@ -113,15 +113,18 @@ def main():
             results['status issues'] = status_issues
 
         # Inspect files
+        query = "/search/?type=File&dataset=/experiments/"+obj['accession']+'/'
```

Comment:
Think we can use @id here and string formatting. 
`query = '/search/?type=File&dataset={}'.format(obj['@id'])`